superset/connectors/base/models.py

```diff
@@ -61,6 +61,7 @@ class BaseDatasource(AuditMixinNullable, ImportMixin):
     cache_timeout = Column(Integer)
     params = Column(String(1000))
     perm = Column(String(1000))
+    schema_perm = Column(String(1000))
```

Comment:
Does this have any context for the Druid NoSQL connector? If not shouldn't this reside on the `tables` table?

Reply:
yep it works for both Druid and SQLatable.